src/Project/ToolRunner.ts

```diff
@@ -50,7 +51,7 @@ export class ToolRunner {
       Logger.append(data.toString());
     });
 
-    this.child!.on(K_EXIT, (code: number|null, signal: NodeJS.Signals|null) => {
+    this.child!.on(K_EXIT, (code: ExitCode|null, signal: NodeJS.Signals|null) => {
```

Comment:
`number` is changed to `string`, is it intended?

Reply:
Yes. For the reason in https://github.com/Samsung/ONE-vscode/pull/749#discussion_r888612720.
While looking at the code,
```
private onFinishedRunningOnecc(val: Number|SelfKilled) {
```

I thought that `Number` is not very specific and hard to figure out what kinda number should be passed.
So I decided to be more specific for clear understanding by changing `Number` to `ExitCode`.

```
private onFinishedRunningOnecc(val: ExitCode|SelfKilled) {
```

I tried to do this change in this PR since this change was related to signature change introduced with `SelfKilled`.